ts/Switch/index.tsx

```diff
@@ -1,16 +1,18 @@
 import {h} from 'preact';
 import MaterialComponent from '../MaterialComponent';
 
-/**
- * @prop disabled = false
- */
-export default class Switch extends MaterialComponent {
-  constructor() {
-    super();
-    this.componentName = 'switch';
-    this._mdcProps = ['disabled'];
-  }
-  materialDom(allprops) {
+export interface ISwitchProps extends JSX.HTMLAttributes {}
```

Comment:
shouldn't disabled be a prop?

Reply:
Can explicitly declare it, but it's included through `JSX.HTMLAttributes`, which is included via `MaterialComponent`